tests/jvm/src/test/scala-2.12/scala/meta/tests/semanticdb/MemberSuite.scala

```diff
@@ -160,4 +160,42 @@ class MemberSuite extends DatabaseSuite(SemanticdbMode.Slim, MemberMode.All) {
     """.stripMargin
   )
 
+  members(
+    """
+      |object a {
+      |  val run = new java.lang.Runnable {
+      |    def run(): Unit = ()
+      |  }
+      |  val timer = new java.util.TimerTask {
```

Comment:
Is `val timer = ` required? What happens if it's a `def` or just `new TimerTask`?

Reply:
not required, removing